tests/streaming_tests.py

```diff
@@ -8,42 +8,23 @@
 
 import pytest
 
-from nwbinspector import Importance, InspectorMessage, inspect_all
-from nwbinspector.testing import check_hdf5_io_open, check_streaming_tests_enabled
-from nwbinspector.tools import read_nwbfile
+from nwbinspector import Importance, InspectorMessage, inspect_dandiset
+from nwbinspector.testing import check_streaming_tests_enabled
 from nwbinspector.utils import FilePathType
 
 STREAMING_TESTS_ENABLED, DISABLED_STREAMING_TESTS_REASON = check_streaming_tests_enabled()
```

Comment:
I think `check_streaming_tests_enabled` needs to be updated to not require that ROS3 driver is installed - the streaming tests in the workflows are being skipped.

Reply:
Good catch - fixed